index.html

```diff
@@ -22,9 +22,24 @@ <h2 class="info-note highlighted-text">You can check out all our events below!!<
     <h3 class="info-note"> To know how this page is getting developed, join our Discord Channel. Contact one of the existing members </h3>
   </main>
 
+  <div class="subscribe">
+      <div class="subscribe__heading">
+        <span>Want to stay updated?</span>
+        <span>Subscribe to  our Newsletter</span>
+      </div>
+      <div class="subscribe__inner">
+        <input class="subscribe__input" placeholder="Enter Email address" />
+        <button class="subscribe__button">Subscribe</button>
+      </div>
+      <div class="subscribe_status"></div>
+  </div>
```

Comment:
I wanted a drop-in JS that created this on document ready event. Please create that.